ci-operator/README.md

```diff
@@ -5,6 +5,20 @@ This document describes how to create CI jobs for Openshift components using
 ci-operator and is intended for component developers who want to add tests to
 their CI process.
 
+After editing the files under this directory, make sure to run the generator to
+ensure that your changes are compliant to our conventions and pass the CI tests
+that will run when you submit your changes as a PR:
+
+```
+docker pull registry.svc.ci.openshift.org/ci/ci-operator-prowgen:latest
+docker run -it -v $(pwd)/ci-operator:/ci-operator:z           \
+  registry.svc.ci.openshift.org/ci/ci-operator-prowgen:latest \
+  --from-dir /ci-operator/config/ --to-dir /ci-operator/jobs
+```
```

Comment:
Instructing people to run the generator is one thing, but we maybe we should advise them what is the expected result to pass the CI later.

Reply:
What do you mean?

Reply:
I am an engineer trying to follow these instructions. I make some changes and run dockerized generator. Right now, it spits out ~40 warnings about name length you don't need to care about (side note: I now think we should remove these). The CI will pass if 1) generator does not hit any error, 2) the generator does not generate anything that differs from the committed state. We can expect people to find out 1) is needed, but not 2). So we should tell them.

"Running the generator over your commits should not further change any of the files under `ci-operator/jobs`; if it does, the CI will fail." or something similar.